packages/validator/src/lib/psi/PsiResult.ts

```diff
@@ -30,6 +30,20 @@ export type LighthouseCategories = {
   [key in LighthouseCategoryName]: LighthouseCategory;
 };
 
+export type LighthouseAuditDetailsItem = {
+  [key: string]: number;
```

Comment:
Can we not just hard code the keys we care about like `firstContentfulPaint` and `largestContentfulPaint`? I feel that would make the code easier to understand.

Reply:
Added
```typescript
export type LighthouseMetricAuditNames = 'firstContentfulPaint' | 'largestContentfulPaint' |
    'maxPotentialFID' | 'cumulativeLayoutShift';
```